### DISCUSSION TITLE
SVG file is unclear

### DISCUSSION THREAD

**Discussion by I-AM-A-NOOB** [d2phap/ImageGlass#1414] [General]
When I opened an _**SVG**_ file, the file renders **unclearly**, like a **bitmap**. However when I tried to open the file on Firefox, it is clear.
Sorry for my bad English, I am not from an English-speaking country

**Comment by d2phap** [comment]
ImageGlass does not support scaling SVG yet but there is a plan for it in v9.0
https://github.com/d2phap/ImageGlass/issues/421

**Reply by Lemonlee8** [reply]
Great，I'm expecting the support for scaling **SVG**, too. The other one is **EPS** file.  Thanks a lot.
